tests/unit-tests/api/utilities/Tests_Functions.php

```diff
@@ -0,0 +1,497 @@
+<?php
```

Comment:
Add simple PHPDoc for class and methods in this file to replace non WP standards compliant comments. Example:

```
/***********************************************
 * Tests for beans_render_function()
 **********************************************/
```

I would be in favor of using the PHPUnit `@covers` tag too (see [here](https://phpunit.de/manual/3.7/en/appendixes.annotations.html#appendixes.annotations.covers)).

Reply:
I often use this technique to provide a grouping comment for like types of tests, methods, functionality, etc.  It is not tied to a single method, but rather a group of methods.  It provides a quick visual to say "Hey, this type of comment means everything from here down ....or until the next one like this....are grouped together."